packages/p2p/src/hooks/use-p2p-rendered-adverts.ts

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import { isMobile } from '@deriv/shared';
```

Comment:
just as a general consensus, can we use the is_mobile value instead? better we start using that as we create new code or update any files 👍